### PR TITLE
Create decision record (ADR) for using render

### LINKED ISSUE #1: Create decision record for deployment
Use the `decisions/_template.qmd` file as a starting point. More or less, move seedcase-project/seedcase-sprout#5 into this record.

### PR DESCRIPTION
**Pull request by philter87**
Closes #1

### CODE REVIEW

**Review by signekb — DISMISSED** [review]
Nice! Just have some minor comments :)

File: decisions/why-render/index.qmd [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+| Github Actions Integration                                      | Very good      | Sort of (deploy webhook)     | Yes                      | Yes                       |
+| Managed Postgres                                                | Yes            | Yes                          | Yes                      | Yes                       |
+| File Storage service                                            | Yes            | Yes (FTP) 0.25$/GB           |                          |                           |
+| "Native support for Django app"                                 | Sort of        | Yes<br>(Poetry support)      | ?                        | Yes<br>(requirements.txt) |
+| Docker Image or Dockerfile support                              | Yes            | Yes                          | Yes                      | No                        |
+| Docker-compose                                                  | Yes            | No                           | Yes (with a normal node) | No                        |
```
[the remark this answers is not in the document]
Comment (philter87) [reply decisions/why-render/index.qmd]: You are right. I will add it earlier when I list the features/criterias

File: decisions/why-render/index.qmd [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Managed Postgres. Cloud provider should have a managed postgres database to store application data if needed
+- "Native" support for Django application. Cloud provider should have support for running a Django application without the need to wrap it in package or docker container
+- Docker image or Dockerfile support. Cloud provider should also have support for running docker images. Potentially, providing a Dockerfile would be convenient
+- Logging + Alerting. Logging and alert is nice, but no so relevant for a demo environment
+- Similar to production environment. We want the demo environment to be similar to the production environment, but we dont know how that will look
+
+
+## Considered Options
+
+We have considered four different cloud providers: Azure, Render, Digital Ocean and Vercel. Below you see a "decision matrix" based on the different features.
```
[the remark this answers is not in the document]
Comment (philter87) [reply decisions/why-render/index.qmd:34]: Yes, you are sortof right, but there are just a bit too many sources and some is also trial-and-error

File: decisions/why-render/index.qmd [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Furthermore, we can customize the solution to use a Docker image or a Dockerfile if we need to. 
+
+The other cloud providers, where not choosen for these reasons:
+- Azure is able to do everything, but it is too complicated for a demo environment
+- Vercel is easy to use, but lacked customisability. For instance, it seemed not to work so well with poetry
+- Digital Ocean could be a fair choice, but I was unable to make it work
+
+### Consequences
+
+Render.com does not understand docker-compose files which would have been nice. Azure and Digital Ocean sort of supports docker-compose files, but it would require a lot of work and this seems overkill for a demo environment
```
[the remark this answers is not in the document]
Comment (philter87) [reply decisions/why-render/index.qmd]: Good idea. I will state it earlier in the requirements list

**Review by K-Beicher — DISMISSED** [review]
Looks good.  I think a suitable test of usability is if an IT professional can't get it to work then the rest of the our users haven't got a snowball's chance in hell to do so :-)

**Review by lwjohnst86 — APPROVED** [review]
Made some edits, looks good thanks!!! :star_struck:

**Pull request merged**